src/comm/ModDevPoll.cc

```diff
@@ -48,19 +45,15 @@
 
 #define DEBUG_DEVPOLL 0
 
-// OPEN_MAX is defined in <climits>
-#define DEVPOLL_UPDATESIZE  OPEN_MAX
-#define DEVPOLL_QUERYSIZE   OPEN_MAX
+// Maximum number of fd's to buffer requests to change polling state
+#define DEVPOLL_UPDATESIZE  256
+// Maximum number of fd's to receive polling updates for in a single syscall
+#define DEVPOLL_QUERYSIZE   1024
```

Comment:
I'm not an expert either, but I don't know of any reasons beyond attempting to optimize for memory usage over number of system calls/loop iterations made.  I'll defer to your judgement here.

Reply:
Thank you. Since this is not a performance optimization PR, we do not need to make complex performance optimization decisions. Let's keep it simple for now.